### ISSUE TITLE
When user double tap the Gender field - drop down options are not visible

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#351]
**Describe the bug**
When the user double-taps the Gender field - drop-down options are not visible- user needs to double tap on the dropdown arrow to get gender options

**Expected result**
When the user double-taps the Gender field - drop-down options should be visible - user doesn't need to double-tap on the dropdown arrow to get gender options

**To Reproduce**
1. Talkback is ON
2. On the user registration page 
3. User enter first name and last name fields
4. When the user clicks on the gender field - gender options did not pop up

https://user-images.githubusercontent.com/78338755/176382129-561d1f96-0fce-4c91-9e87-3811d4519781.mp4

**Comment by mikedawson** [comment]
HI @poojaustad I think this should be resolved in the MVVM release using Jetpack compose / React, can you please check / close if it is resolved?

**Comment by poojaustad** [comment]
REsolved
